src/main/java/appeng/tile/crafting/TileCraftingTile.java

```diff
@@ -99,9 +108,20 @@ public boolean isAccelerator() {
                 .contains(this.worldObj.getBlockMetadata(this.xCoord, this.yCoord, this.zCoord) & 3);
     }
 
+    public int getAcceleratorScaleFactor(Block blockType) {
+        if (blockType instanceof BlockAdvancedCraftingUnit) {
+            return ACCELERATOR_SCALE_FACTOR * 16;
+        } else {
+            return ACCELERATOR_SCALE_FACTOR;
+        }
+    }
+
     public int acceleratorValue() {
         int meta = this.worldObj.getBlockMetadata(this.xCoord, this.yCoord, this.zCoord) & 3;
-        return (int) Math.pow(ACCELERATOR_SCALE_FACTOR, meta - 1);
+        return (int) Math.pow(
+                this.getAcceleratorScaleFactor(
+                        this.worldObj.getTileEntity(this.xCoord, this.yCoord, this.zCoord).getBlockType()),
```

Comment:
Do `worldObj.getBlock` instead of `worldObj.getTileEntity.getBlockType`

Reply:
Thank you for your advice

Reply:
fixed